clientInit.js

```diff
@@ -0,0 +1,49 @@
+const __controllerTemplate = 'Controller';
+const __appVersion = __ver || 0;
+
+var controller = typeof(controller) === 'undefined' ? null : controller;
+var firstTimeLoad = typeof(firstTimeLoad) === 'undefined';
+
+// Establish a new session whether first time or because of expiry / server restart
+function __bindController (newController, sessionExpiration) {
+    if (controller && typeof(controller.shutdown) === 'function') {
+        controller.shutdown();
+    }
+    controller = newController;
+    console.log('Binding Controller');
+    if (firstTimeLoad) {
+        firstTimeLoad = false;
+        if (typeof(controller.clientInit) === 'function') {
+            controller.clientInit(sessionExpiration);
+        }
+    }
+    else {
+        if (typeof(controller.clientInit) === 'function') {
+            controller.clientInit(sessionExpiration);
+        }
+        controller.refresh(1);
```

Comment:
`refresh()` is a bindster function - do we want to do this if this is meant to be a non-bindster clientInit lib?

Reply:
I see controller.refresh being called throughout the repo. Are you sure this is bindster related? If this is the case, then the __refresh function does nothing. So amorphic's client.js should not even have it as an argument in establishClientSession, or use it at lines 174 and 285